src/libs/webpage-utils.ts

```diff
@@ -0,0 +1,99 @@
+import * as cheerio from "cheerio";
+import fetch, { Response } from "node-fetch";
+import { Document } from "src/elastic";
+import { DrupalSettingsJson } from "@types";
+
+/**
+ * Gets element from pages head
+ * 
+ * @param pageResponse Response
+ * @param elementName head element name
```

Comment:
elementName is a bit misleading when it's actually used as selector

Reply:
Resolved